app/jobs/ai_oa_status_export_job.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class AiOAStatusExportJob < ApplicationJob
+  queue_as 'default'
+
+  def perform
```

Comment:
We can pass the file id in here as an argument.  Then, `find` the file below before passing it into the exporter.